src/packer.pkr.hcl

```diff
@@ -94,9 +93,8 @@ source "amazon-ebs" "windows" {
   kms_key_id                  = var.build_region_kms
   launch_block_device_mappings {
     delete_on_termination = true
-    device_name           = "/dev/xvda"
-    encrypted             = true
-    volume_size           = 8
+    device_name           = "/dev/sda1"
+    volume_size           = 1000
```

Comment:
Does this really need to be a terabyte? Why did you remove encryption?

Reply:
I changed both the volume size and encryption settings while debugging issues with the packer build. Thank you for catching this.

Reply:
Addressed here https://github.com/cisagov/windows-commando-vm-packer/pull/21/commits/97492332557b842a335dab40588fc75e2704f82b